src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -1,16 +1,28 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import jakarta.persistence.EntityNotFoundException;
+import mate.academy.hibernate.relations.dao.CountryDao;
+import mate.academy.hibernate.relations.dao.impl.CountryDaoImpl;
 import mate.academy.hibernate.relations.model.Country;
 import mate.academy.hibernate.relations.service.CountryService;
+import org.hibernate.SessionFactory;
 
 public class CountryServiceImpl implements CountryService {
-    @Override
+    private CountryDao countryDao;
+    private SessionFactory sessionFactory;
+
+    public CountryServiceImpl(SessionFactory sessionFactory) {
+        this.sessionFactory = sessionFactory;
+        countryDao = new CountryDaoImpl(sessionFactory);
+    }
+
```

Comment:
Service does not need SessionFactory 
```suggestion
    private CountryDao countryDao;

    public CountryServiceImpl(CountryDao countryDao) {
        this.countryDao= countryDao;
    }
```